config/config.go

```diff
@@ -88,6 +90,7 @@ func NewDefaultConfig() *Config {
 			RetryCount: -1,
 		},
 		Cache: CacheConfig{
+			Enabled:  true,
```

Comment:
@burn2delete - I believe this is expected behavior, but was unclear so this just explicitly states it. lmk if this is incorrect- can fix